app/code/Magento/Customer/Controller/Address/Delete.php

```diff
@@ -6,9 +6,7 @@
  */
 namespace Magento\Customer\Controller\Address;
 
-use Magento\Framework\App\Action\HttpPostActionInterface;
-
-class Delete extends \Magento\Customer\Controller\Address implements HttpPostActionInterface
+class Delete extends \Magento\Customer\Controller\Address
```

Comment:
Removing the interface is backward incompatible change. That might result in error on instances where this class is passed to a method that require `HttpPostActionInterface` parameter

Reply:
So, regarding this, do you think i have to report a new bug issue? i mean, i removed the interface, couse the current storefront use GET method (https://github.com/magento/magento2/blob/2.3-develop/app/code/Magento/Customer/view/frontend/web/js/address.js#L65), or should i fix it with this PR?

Reply:
Using the controller to retrieve information via GET is correct behaviour and I believe the functionality will continue working even if Controller implements HttpPostActionInterface, however, removing an interface implementation is not backward compatible, so I would preserve it.

You could actually make this controller implement both HttpPostActionInterface and HttpGetActionInterface to mark it as get action